master/docs/developer/master-worker.rst

```diff
@@ -39,7 +39,8 @@ Bot methods
 The worker-side Bot object has the following remote methods:
 
 :meth:`~buildbot_worker.pb.BotPb.remote_getCommands`
-    Returns a list of ``(name, version)`` for all commands the worker recognizes.
+    Returns a dictionary ``Dict[name, version]`` for all commands the worker recognizes.
```

Comment:
What was meant here is that the result is Python `dict` with names as the keys, and versions as the values, yes.
I used notation of [PEP 484](https://www.python.org/dev/peps/pep-0484/) which might be used with `typing` module in Python 3.

I'll replace this with explanation in English.